renderer/brave_content_settings_observer_helper.h

```diff
@@ -12,7 +12,9 @@
 #include "third_party/blink/renderer/platform/bindings/script_state.h"
 
 static bool AllowFingerprinting(blink::LocalFrame* frame) {
-  if (!frame) return true;
+  if (!frame || !frame->GetContentSettingsClient()) {
```

Comment:
is it possible for `GetContentSettingsClient` to ever be null? Maybe during tab close or shutdown? If not that's fine, just want to be sure.